src/styles/mediaQuery.ts

```diff
@@ -0,0 +1,26 @@
+const breakPoints: BreakPointMap = {
+  xs: 0,
+  sm: 576,
+  md: 768,
+  lg: 992,
+  xl: 1200,
+  xxl: 1400,
+};
+
+export type BreakPoint = "xs" | "sm" | "md" | "lg" | "xl" | "xxl";
```

Comment:
This seems redundant, since these strings are already defined in breakPoints. How about something like

type BreakPointMap = { [key: string]: number };

export type BreakPoint = keyof typeof breakPoints;